synapse/docker/synapse_dockerfile

```diff
@@ -21,26 +21,22 @@ RUN pip3 install --upgrade\
     setuptools\
     wheel
 
-# Set the locale
-RUN locale-gen en_US.UTF-8
+# set the locale
+RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen &&\
+    locale-gen en_US.UTF-8 &&\
+    dpkg-reconfigure locales &&\
+    /usr/sbin/update-locale LANG=en_US.UTF-8
 ENV LANG en_US.UTF-8
-ENV LANGUAGE en_US:en
+ENV LANGUAGE en_US.UTF-8
 ENV LC_ALL en_US.UTF-8
 
-RUN mkdir ~/git &&\
- mkdir /syndata
+# setup synapse
+COPY . /root/git/synapse
+RUN mkdir /syndata &&\
+    cd /root/git/synapse &&\
```

Comment:
Since having more than one RUN directive seems like no biggie ( noticed another below ) could we remove the ugly \ continuations by splitting this into 2:
```
RUN mkdir /syndata
RUN cd /root/git/synapse && python3 setup.py develop
```